### ISSUE TITLE
disable global transitions

### ISSUE THREAD

**Issue by donovan-graham** [ef4/liquid-fire#275]
I've recently upgraded to use liquid-fire@0.19.2 with ember@1.11.1  there now seems to be a global slideDown transition applied to all routes.  I only need transitions on specific routes, so how do I disable this global behaviour?

Many thanks. D

**Comment by ef4** [comment]
Hi, there is no intentional global slide down transition. It seems more likely that one of your own rules is now accidentally matching too broadly. If you post your transition map I can try to see which.

**Comment by donovan-graham** [comment]
Hi

Here's my transition map.

```
export default function() {
  this.transition(
    this.fromRoute('login'),
    this.toRoute('forgot-password'),
    this.use('toLeft'),
    this.reverse('toRight')
  );

  this.transition(
    this.fromRoute('login'),
    this.toRoute('register'),
    this.use('toLeft'),
    this.reverse('toRight')
  );
```

**Comment by ef4** [comment]
Nothing there looks wrong.

Maybe you're actually seeing the liquid-outlet's default growth animation? There was a bug that was causing `enableGrowth=false` to be ignored, it's fixed in 0.19.4.

**Comment by donovan-graham** [comment]
I've made a brand new app with 2 routes and no transition map.

**Comment by ef4** [comment]
Yes, you need enableGrowth=false, and that's currently broken in 0.19.x, fixed in 0.20.x. I'm also willing to release a backport in the 0.19.x series for people who aren't ready to go to glimmer, but nobody has found the time to do that yet.

**Comment by winne42** [comment]
@ef4 I suppose that this ticket is severely outdated by now and can be closed?
